### PR TITLE
Used minimum heap size for images. Updated libraries. Removed redundant env variables. Used micronaut base image. Remove db from view.

### PR DESCRIPTION
**Pull request by dartartem**

### CODE REVIEW

File: customers-service/Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
-FROM java:openjdk-8u111-alpine
-RUN apk --no-cache add curl
+FROM openjdk:8u252-slim-buster
```
Comment (cer) [inline customers-service/Dockerfile:1]: Please use base image - BUILD-5

File: docker-compose-eventuate-local-mysql.yml [hunk 2]
```diff
@@ -1,21 +1,28 @@
...
     ports:
       - 2181:2181
-
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
+      KAFKA_HEAP_OPTS: -Xmx64m
 
   kafka:
-    image: eventuateio/eventuate-kafka:$EVENTUATE_KAFKA_VERSION
+    image: "confluentinc/cp-enterprise-kafka:5.2.4"
```
Comment (cer) [inline docker-compose-eventuate-local-mysql.yml:12]: Don't use enterprise - just cp-kafka

File: set-env.sh [hunk 3]
```diff
@@ -14,4 +14,5 @@ export COMPOSE_HTTP_TIMEOUT=240
 export EVENTUATELOCAL_KAFKA_BOOTSTRAP_SERVERS=$DOCKER_HOST_IP:9092
 export EVENTUATELOCAL_CDC_DB_USER_NAME=root
```
Comment (cer) [inline set-env.sh:15]: Are any of these env vars necessary?
In the SPring application the application.properties use `${DOCKER_HOST_IP:localhost}`

Comment (dartartem) [reply set-env.sh:15]: EVENTUATELOCAL_KAFKA_BOOTSTRAP_SERVERS - probably used by tests.
COMPOSE_HTTP_TIMEOUT - sometimes ago solved some issue, I don't remembed what it was

File: docker-compose-eventuate-local-mysql.yml (hunk 2 above)
Comment (dartartem) [reply docker-compose-eventuate-local-mysql.yml:12]: ok

File: customers-service/Dockerfile (hunk 1 above)
Comment (dartartem) [reply customers-service/Dockerfile:1]: ok

**Comment by cer** [comment]
Fixes https://github.com/eventuate-examples/eventuate-micronaut-examples-customers-and-orders/issues/8

File: customers-service/Dockerfile [hunk 4]
```diff
@@ -1,5 +1,4 @@
-FROM java:openjdk-8u111-alpine
-RUN apk --no-cache add curl
-CMD java ${JAVA_OPTS} -jar customers-service.jar
-HEALTHCHECK --start-period=30s --interval=5s CMD curl -f http://localhost:8080/health || exit 1
-COPY build/libs/customers-service.jar .
+ARG baseImageVersion
+FROM eventuateio/eventuate-examples-docker-images-java-example-base-image:$baseImageVersion
+COPY build/libs/customers-service.jar service.jar
+HEALTHCHECK --start-period=30s --interval=5s CMD curl -f http://localhost:8080/health || exit 1
```
Comment (cer) [inline customers-service/Dockerfile:4]: There is a micronaut version of the base image - BUILD-6

Comment (dartartem) [reply customers-service/Dockerfile:4]: I will update

**Pull request merged**